### ISSUE TITLE
Generate basic report with out meme comparison

### ISSUE THREAD

**Issue by kaillahs** [jmschrei/tfmodisco-lite#54]
Hi! I am using tf-modisco lite on data generated using chrombpnet. I generated the modisco.h5 file and want to now generate the report. Is it still possible to generate the report using the command `modisco report -i modisco_results.h5 -o report/ -s report/` without the `-m meme.txt` input or has that option been omitted (I am getting an error message alerting me that the pipeline is missing the meme.txt input)?  Thank you in advance for your help!

**Comment by jmschrei** [comment]
Yes, you shouldn't need the MEME file if you don't want to map your learned motifs to a database. Have you checked to make sure you are using the latest version? Can you post the entire command you used and the error message you got?

**Comment by kaillahs** [comment]
Hi Jacob - thank you for your quick reply. I actually moved all of my work to a different PC and the error
 doesn't seem to be occurring anymore! However, the motif.html that it generated is not able to load any of the cwm images...

![Screenshot 2024-08-07 at 12 32 23 PM](https://github.com/user-attachments/assets/d2c12526-03bf-4f9e-aa35-3a6a299e4bc6)

**Comment by kaillahs** [comment]
The above report was for a specific region on using the chromBPNet tutorial model. I just tried using tfmodisco-lite on my own chrombpnet model (still using the tutorial bias model), but the report isn't showing any motifs this time. Any idea why this may be happening?

**Comment by jmschrei** [comment]
Are you running the report command in the same directory you're running the motifs command? What's in the `report/` directory after running it?

**Comment by kaillahs** [comment]
Yes, I am running both commands in the same directory. The report folder contains an empty trimmed_logos folder as well as a motifs HTML file.

**Comment by kaillahs** [comment]
I generated another run with a different region and got 4 patterns, however, the same issue is occurring where the images are not fully showing up in the HTML despite being in the trimmed_logos folder.

**Comment by kaillahs** [comment]
Hi @jmschrei - I am checking in to see if you have any suggestions to resolve the above-mentioned issues... I've been running other regions through the pipeline, all of which are successfully creating pred_bw files that can be viewed through IGV, and a lot of them are not generating any motifs. Thank you in advance!

**Comment by jmschrei** [comment]
Hi @kaillahs. Unfortunately, issues like this are usually challenging to debug remotely. But what do you mean the report is generated for a specific region? Are you running tfmodisco on a single example? The report should be generated genome-wide. I do not believe that there are over 100 real seqlets in a single example.

**Comment by kaillahs** [comment]
@jmschrei  - thank you for your reply. I've been using the chrombpnet contribs_bw pipeline which requires you to enter a specific region of the genome you want to run through the command. I've been then using the output_prefix.profile_scores.h5 and the output_prefix.counts_scores.h5 for the -i argument of the modisco motifs command. Please correct me if this is the wrong input.

**Comment by jmschrei** [comment]
Sorry, but I'm actually unfamiliar with how the chrombpnet repo works. @austintwang do you happen to know?

**Comment by panushri25** [comment]
@kaillahs 3000 bp is a very small region to get deepshaps and summarize seqlets using modisco. What are you trying to do?

Modisco is a summarization tool used to get a list of motifs found in all your peaks for example.

**Comment by kaillahs** [comment]
@panushri25 I am trying to get transcription factors across specific peak regions in the genome. These regions tend to be fairly small... Would it be best to just extend the region to around 300,000 bp? I've done this once and it seems to work just fine so I'm assuming the issues I was running into were simply size-related.

**Comment by kaillahs** [comment]
Hi! I am noticing that my match q-values are far lower for the counts score modisco reports than for the profile scores. Is there a reason for this? Also, should there be a difference in the quality of the modiso motif predictions depending on how far out I extend the regions? I seem to be getting around 8 predictions regardless of whether I extend my region by 360,000bp or 150,000bp, 4 of which are present in both for the count predictions and 1 of which is present in both for profile prediction.

**Comment by austintwang** [comment]
Hi, tfmodisco-lite has been merged into the [official TF-MoDISco repository](https://github.com/kundajelab/tfmodisco). We encourage you to re-open your issue there if it hasn't been resolved.
